cmd/mesh_bridge/main.go

```diff
@@ -0,0 +1,106 @@
+// +build !js
+
+// mesh_bridge is a short program that bridges two Mesh nodes. This is useful in cases where
+// we introduce a network-level breaking change but still want the liquidity from one network
+//to flow to another
+package main
+
+import (
+	"context"
+	"fmt"
+	"sync"
+
+	"github.com/0xProject/0x-mesh/rpc"
+	"github.com/0xProject/0x-mesh/zeroex"
+	"github.com/plaid/go-envvar/envvar"
+	log "github.com/sirupsen/logrus"
+)
+
+type clientEnvVars struct {
+	FirstWSRPCAddress  string `envvar:"FIRST_WS_RPC_ADDRESS"`
+	SecondWSRPCAddress string `envvar:"SECOND_WS_RPC_ADDRESS"`
+	Verbose            bool   `envvar:"VERBOSE"`
+}
+
+func main() {
+	env := clientEnvVars{}
+	if err := envvar.Parse(&env); err != nil {
+		panic(err)
+	}
+
+	if env.Verbose {
+		log.SetLevel(log.Level(4))
+	} else {
+		log.SetLevel(log.Level(0))
+	}
+
+	firstClient, err := rpc.NewClient(env.SecondWSRPCAddress)
+	if err != nil {
+		log.WithError(err).Fatal("could not create client")
+	}
+	stats, err := firstClient.GetStats()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.WithField("stats", stats).Info("Spun up first client")
+
+	secondClient, err := rpc.NewClient(env.FirstWSRPCAddress)
+	if err != nil {
+		log.WithError(err).Fatal("could not create client")
+	}
+	stats, err = secondClient.GetStats()
+	if err != nil {
+		log.Fatal(err)
+	}
+	log.WithField("stats", stats).Info("Spun up second client")
+
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		pipeOrders(secondClient, firstClient, "v8", "v9")
+	}()
+	wg.Add(1)
+	go func() {
+		defer wg.Done()
+		go pipeOrders(firstClient, secondClient, "v9", "v8")
```

Comment:
Should this be a go func? If it should, should the other `pipeOrders` be called from a go func?

Reply:
We're already in a goroutine so we don't need to start another one.

```suggestion
		pipeOrders(firstClient, secondClient, "v9", "v8")
```